dumper/app/src/main/resources/oracle-stats/awr/hist-cmd-types-awr.sql

```diff
@@ -18,16 +18,16 @@ SELECT
   to_char(C.begin_interval_time, 'hh24') "Hour",
   D.name "CommandName",
   count(1) "Count",
-  sum(A.buffer_gets_total) "SumBufferGets",
-  sum(A.elapsed_time_total) "SumElapsedTime",
-  sum(A.rows_processed_total) "SumRowsProcessed",
-  sum(A.executions_total) "SumExecutions",
-  sum(A.cpu_time_total) "SumCpuTime",
-  sum(A.iowait_total) "SumIOWait",
-  sum(A.clwait_total) "SumCLWait",
-  sum(A.apwait_total) "SumAPWait",
-  sum(A.ccwait_total) "SumCCWait",
-  sum(A.plsexec_time_total) "SumPLSExecTime"
+  avg(A.buffer_gets_total) "AvgBufferGets",
```

Comment:
We should use `_delta` columns here. Please check the original version: https://github.com/GoogleCloudPlatform/database-assessment/blob/main/scripts/collector/oracle/sql/extracts/awr/awrhistcmdtypes.sql

Reply:
Done